Jenkinsfile

```diff
@@ -28,8 +28,8 @@ stage('Checkout') {
       versionSuffix = "alpha.${env.BUILD_ID}"
     }
     // TODO: temp for beta releases
-    else if (env.CHANGE_BRANCH != null && env.CHANGE_BRANCH == "release/10.2.0-beta.1") {
-      versionSuffix = "beta.1"
+    else if (env.CHANGE_BRANCH != null && env.CHANGE_BRANCH == "release/10.2.0-beta.2") {
```

Comment:
This change isn't related. If it's here by mistake, then I made you notice it. Otherwise, disregard this comment.

Reply:
Yes - I forgot to include it in my previous PR, so figured might as well add it here 🤦‍♂️